src/reference/changes.rst

```diff
@@ -0,0 +1,91 @@
+Changes
+=======
+
+.. _cylc-flow-changelog: https://github.com/cylc/cylc-flow/blob/master/CHANGES.md
+.. _cylc-uiserver-changelog: https://github.com/cylc/cylc-uiserver/blob/master/CHANGES.md
+.. _cylc-ui-changelog: https://github.com/cylc/cylc-ui/blob/master/CHANGES.md
+.. _cylc-rose-changelog: https://github.com/cylc/cylc-rose/blob/master/CHANGES.md
+.. _metomi-rose-changelog: https://github.com/metomi/rose/blob/master/CHANGES.md
+.. _metomi-isodatetime-changelog: https://github.com/metomi/isodatetime/blob/master/CHANGES.md
+
+This page contains a summary of significant changes across all Cylc components for each
+release.
+
+For more detail see the component changelogs:
+
+* `cylc-flow-changelog`_
+* `cylc-uiserver-changelog`_
+* `cylc-ui-changelog`_
+* `cylc-rose-changelog`_
+* `metomi-rose-changelog`_
+* `metomi-isodatetime-changelog`_
+
+----------
+
+Cylc 8.1.0
+----------
+
+.. TODO: updade me before release:
+
+   .. admonition:: Cylc Components
+      :class: hint
+
+      TODO: fill in component versions
+
+Cylc Graph
+^^^^^^^^^^
+
+The web UI now has a graph view which displays a visualisation of a workflow's graph:
+
+.. image:: changes/cylc-graph.gif
+   :width: 80%
+
+Family & cycle grouping as well as the ability to view graphs for stopped workflows
+will be added in later releases.
+
+Combined Commands
+^^^^^^^^^^^^^^^^^
+
+A new command called ``cylc vip`` which validates, installs, then plays a workflow
+as a convenient alternative to typing:
+
+.. code-block:: bash
+
+   cylc validate <path>
+   cylc install <path>
+   cylc play <id>
+
+.. image:: changes/cylc-vip.gif
+   :width: 80%
+
+Bash Completion
+^^^^^^^^^^^^^^^
+
+Cylc now provides a high performance Bash completion script which saves you typing:
+
+* Cylc commands & options
+* Workflow IDs
+* Cycle points
+* Task names
+* Job numbers
+
+.. image:: changes/cylc-completion.bash.gif
+   :width: 80%
+
+:ref:`Installation instructions <installation.shell_auto_completion>`.
+
+----------
+
+Cylc 8.0.0
+----------
+
+.. admonition:: Cylc Components
+   :class: hint
+
+   :cylc-flow: `8.0 <https://github.com/cylc/cylc-flow/blob/8.0.0/CHANGES.md#major-changes-in-cylc-8>`_
+   :cylc-uiserver: `1.1 <https://github.com/cylc/cylc-uiserver/blob/1.1.0/CHANGES.md#cylc-uiserver-110-released-2022-07-28>`_
+   :cylc-rose: `1.1 <https://github.com/cylc/cylc-rose/blob/1.1.0/CHANGES.md#cylc-rose-110-released-2022-07-28>`_
```

Comment:
The tests fail because the target name `1.1` is not unique. Apparently using two trailing underscores rather than one [makes the reference anonymous](https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557)

```suggestion
   :cylc-flow: `8.0 <https://github.com/cylc/cylc-flow/blob/8.0.0/CHANGES.md#major-changes-in-cylc-8>`__
   :cylc-uiserver: `1.1 <https://github.com/cylc/cylc-uiserver/blob/1.1.0/CHANGES.md#cylc-uiserver-110-released-2022-07-28>`__
   :cylc-rose: `1.1 <https://github.com/cylc/cylc-rose/blob/1.1.0/CHANGES.md#cylc-rose-110-released-2022-07-28>`__
```